WooCommerce/Classes/ViewRelated/Products/ProductsViewController.swift

```diff
@@ -628,6 +634,24 @@ private extension ProductsViewController {
     }
 }
 
+// MARK: - Filter UI Helpers
+//
+private extension ProductsViewController {
+    func updateFilterButtonTitle(filters: FilterProductListViewModel.Filters) {
+        let activeFilterCount = FilterProductListViewModel(filters: filters).numberOfActiveFilters
```

Comment:
yea I like it! the only thing is that the `numberOfActiveFilters` could be anything to the initializer instead of a derived value as in the commit below (I added another `init()` for initializing without any filters) but it's simpler for the consumer of `Filters` (Products tab view controller) 🙂 

https://github.com/woocommerce/woocommerce-ios/pull/2246/commits/063ee692450d4a86322735db72e0d4d746955f4f

Reply:
That's true. We might get into trouble once there are other sources of “active” filters (not from `FilterProductListViewModel`). For now, we can trust that `FilterProductListViewModel` gives us the correct value.